backend/src/main/java/org/example/restaurantmanagementapplication/rest/BillController.java

```diff
@@ -0,0 +1,59 @@
+package org.example.restaurantmanagementapplication.rest;
+
+import lombok.RequiredArgsConstructor;
+import org.example.restaurantmanagementapplication.mapper.BillMapper;
+import org.example.restaurantmanagementapplication.model.out.BillDto;
+import org.example.restaurantmanagementapplication.service.BillService;
+import org.example.restaurantmanagementapplication.service.OrderService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/bill")
+@RequiredArgsConstructor
+public class BillController {
+
+  private final BillMapper billMapper;
+  private final BillService billService;
+  private final OrderService orderService;
+
+  @RequestMapping("/{orderId}")
```

Comment:
`@RequestMapping` will match any HTTP method, probably should be:
```suggestion
  @GetMapping("/{orderId}")
```

Reply:
My bad, must have run some suggestion